pygerduty/common.py

```diff
@@ -5,15 +5,19 @@
 from .exceptions import Error, IntegrationAPIError, BadRequest, NotFound
 from six import string_types
 from six.moves import urllib
+from random import randint
+from time import sleep
 
 ISO8601_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
 
 
 class Requester(object):
-    def __init__(self, timeout=10, proxies=None, parse_datetime=False):
+    def __init__(self, timeout=10, proxies=None, parse_datetime=False, min_backoff=15.0, max_retries=5):
```

Comment:
Previously the timeout was 10s with no retries; with the retries + backoff it'll be a minimum 75s before the request will give up by default.

I'd prefer to lower the default values (maybe min_backoff=5 and retries=3), and we should provide a way for users to plumb through their own values, here: https://github.com/dropbox/pygerduty/blob/master/pygerduty/v2.py#L611